jscomp/ext/ext_js_suffix.ml

```diff
@@ -1,17 +1,28 @@
-type t = Js | Bs_js | Mjs | Cjs | Unknown_extension
+type t =
+  | Js
```

Comment:
How about listing them in the same order as the `"enum"` list in the change above?

Reply:
Same thing for the 2 functions below.
Easier to check that no case is forgotten.

Reply:
No problem. But I’m not sure I understand what do you mean exactly. This one?

```ocaml
type t =
  | Js
  | Mjs
  | Cjs
  | Bs_js
  | Bs_mjs
  | Bs_cjs
  | Unknown_extension
```

Reply:
> No problem. But I’m not sure I understand what do you mean exactly. This one?
> 
> ```ocaml
> type t =
>   | Js
>   | Mjs
>   | Cjs
>   | Bs_js
>   | Bs_mjs
>   | Bs_cjs
>   | Unknown_extension
> ```

Yes exactly.

Reply:
Done

Reply:
Great, only changelog missing, then can be merged.